src/test/java/core/basesyntax/validation/InputValidatorTest.java

```diff
@@ -0,0 +1,97 @@
+package core.basesyntax.validation;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class InputValidatorTest {
+    private static final int CORRECT_SIZE = 3;
+    private static final int INCORRECT_SIZE = 2;
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+    private static InputValidator inputValidator;
+
+    @BeforeClass
+    public static void beforeAll() {
+        inputValidator = new InputValidatorImpl();
+    }
+
+    @Test
+    public void isValidInput_correctData_Ok() {
+        String[] testData = new String[CORRECT_SIZE];
+        testData[OPERATION_INDEX] = "b";
+        testData[FRUIT_INDEX] = "apple";
+        testData[AMOUNT_INDEX] = "100";
+        inputValidator.isValidInput(testData);
+        Assert.assertTrue("Input data is incorrect!", true);
```

Comment:
Do not hardcode your tests. **_assertTrue()_** should take a result of your method.

Reply:
Stupid mistake, fixed)